app/models/notification.py

```diff
@@ -0,0 +1,16 @@
+from pydantic import BaseModel, Field
+from typing import Dict, Any
+
+
+class BaseModel(BaseModel):
+    def model_dump(self, **kwargs) -> Dict[str, Any]:
+        return super().model_dump(by_alias=True, exclude_none=True, **kwargs)
```

Comment:
We are duplicating this custom `BaseModel` class in every model where we want to enforce serialization settings. We should probably extract this to a generic class (possibly with a more descriptive name) and extend it in the scenarios we need it (currently only used in `profile`).

Reply:
Yes, that could work. It's mostly to avoid duplicating code with this `BaseModel` every time we need a custom serialization for models